### PR TITLE
CF-505 update dgranted cert to be different to granted cert

### PR DESCRIPTION
**Pull request by meyerjrr**

### CODE REVIEW

File: pkg/api/auth_client.go [hunk 1]
```diff
@@ -117,7 +117,11 @@ func GetIdentity() (cert *x509.Certificate, signer crypto.Signer, err error) {
 		if err != nil {
 			return nil, nil, err
 		}
-		if cert.Subject.CommonName == "Granted Identity Certificate" {
+		cert_name := "Granted Identity Certificate"
+		if os.Getenv("GRANTED_DEV") == "true" {
+			cert_name = "Dgranted Identity Certificate"
+		}
+		if cert.Subject.CommonName == cert_name {
```
Comment (chrnorm) [inline pkg/api/auth_client.go:124]: suuuper minor nit
```suggestion
		certName := "Granted Identity Certificate"
		if os.Getenv("GRANTED_DEV") == "true" {
			cert_name = "Dgranted Identity Certificate"
		}
		if cert.Subject.CommonName == certName {
```

File: cmd/cli/commands/certificate.go [hunk 2]
```diff
@@ -18,8 +19,12 @@ var CertificateCommand = cli.Command{
 		}
 		defer store.Close()
 
+		cert_name := "Granted Identity Certificate"
```
Comment (chrnorm) [inline cmd/cli/commands/certificate.go:22]:
```suggestion
		certName := "Granted Identity Certificate"
```

File: cmd/cli/commands/certificate.go [hunk 3]
```diff
@@ -18,8 +19,12 @@ var CertificateCommand = cli.Command{
...
+		if os.Getenv("GRANTED_DEV") == "true" {
+			cert_name = "Dgranted Identity Certificate"
+		}
 		certs, err := store.Certificates(&certstore.CertificateFilter{
-			SubjectStartsWith: "Granted Identity Certificate",
+			SubjectStartsWith: cert_name,
```
Comment (chrnorm) [inline cmd/cli/commands/certificate.go:27]:
```suggestion
			SubjectStartsWith: certName,
```

**Review by chrnorm — CHANGES_REQUESTED** [review]

**Comment by chrnorm** [comment]
A note for your info @meyerjrr - during my initial implementation I had trouble removing old certificates from the keychain. So if we end up with lots of `Dgranted Identity Certificates` after this is merged you might need to take a look in https://github.com/common-fate/certstore

**Review by JoshuaWilkes — APPROVED** [review]

**Pull request merged**